src/test/java/me/itzg/tsdbcassandra/services/DownsampleProcessorTest.java

```diff
@@ -51,8 +51,8 @@
   @TestConfiguration
   static class TestConfig {
     @Bean
-    public Scheduler downsampleScheduler() {
-      return Schedulers.immediate();
+    public TaskScheduler downsampleTaskScheduler() {
+      return new TaskSchedulerBuilder().build();
```

Comment:
there is a big block of commented out code starting on line 196.  when i do that, i generally like to leave a comment for maintainers explaining why it is there and if/when it can be removed.

Reply:
It just needed to be deleted :)